### DISCUSSION TITLE
Open all images with ImageGlass

### DISCUSSION THREAD

**Discussion by santilococo** [d2phap/ImageGlass#791] [General] [answered]
### System information:
 - Windows OS version: 1903
 - ImageGlass version: 7.6

### Questions
Hi, I wanna know if there is a possibilty to open ALL image types (.png, .jpg, among others) with your app. Because I cannot find a config for that.
Greetings!

**Comment by d2phap** [comment, marked as answer]
You can register all supported extensions of ImageGlass by going to Settings > File Type associations, click on `Set as Default photo viewer` button.

After that, open Windows Settings > Apps > Default apps, and select ImageGlass as default photo viewer.

![image](https://user-images.githubusercontent.com/3154213/83935469-66f9f880-a7ec-11ea-92bd-c1cfca9b8f23.png)

**Reply by santilococo** [reply]
Thanks!
